include/fmt/format.h

```diff
@@ -1022,12 +1022,12 @@ template <typename Char> struct fill_t {
 // We cannot use enum classes as bit fields because of a gcc bug
 // https://gcc.gnu.org/bugzilla/show_bug.cgi?id=61414.
 namespace align {
-enum type { none, left, right, center, numeric };
+enum type : uint8_t { none, left, right, center, numeric };
```

Comment:
Reverted because of
```
include/fmt/format.h:1039:19: error: ‘fmt::v6::basic_format_specs<char>::align’ is too small to hold all values of ‘using align_t = enum fmt::v6::align::type {aka enum fmt::v6::align::type}’ [-Werror]
```

Reply:
That's unfortunate; it looks like there isn't a nice way to satisfy both compilers. I can switch the struct data members to use `uint8_t` directly, but then we lose the type safety of enums.